mlir/lib/Dialect/Rock/IR/MfmaInsnGroup.cpp

```diff
@@ -159,153 +159,70 @@ auto getMfmaInsnGroupAttrMap = []() {
   static llvm::DenseMap<MfmaInsnGroupSelectKey, MfmaInsnGroupAttr,
                         MfmaInsnGroupSelectKeyInfo>
       groupAttrMap{{{MfmaTypeId::Fp32TyId, 64, 64},
-                    {ROCDL::mfma_f32_32x32x1f32::getOperationName(),
-                     1,
-                     1,
-                     {{1, 0, MFMAPermB::none}, {1, 1, MFMAPermB::none}}}},
+                    {ROCDL::mfma_f32_32x32x2f32::getOperationName()}},
```

Comment:
This is not a review comment but throwing some thoughts here.

I think compared with CK we have a chance to shorten this list. (But this may moderately impact performance). For example, I think their wave size selection only has "basic units nubmers" like: `64x64`, `32x32`, `16x16` and `4x4`. Then we are pretty much covered with the same support range. I'm not sure how much benefit does the "uneven" cut will bring to us. For example for a block size of 64x32: 
 - We can either work it out using 2 waves of 32x32
 - Or, use one wave of 64x32, but `mRepeat = 2`

In a way, it looks to me now that keeping those entries are still necessary because different topology will yield different solutions.

Reply:
> - We can either work it out using 2 waves of 32x32
> - Or, use one wave of 64x32, but mRepeat = 2

We should be able to achieve that now, but it means having the big table in c++. 

This is what I was saying when I was talking about including `mRepeats`/`nRepeats` in the number of parameters of the tuner. The tuner should decide that, while our table should contain only the unit-tiles (i.e., the building blocks).  

Just to be clear, I am not suggesting to make the search space bigger (unless we benefit from this), but I think the pruning of the search space should be the tuner prerogative and not hardcoded in C++